baseapp_blocks/graphql/object_types.py

```
@@ -61,7 +61,7 @@ class Meta:
         interfaces = (RelayNode,)
 
     @classmethod
-    def get_node(self, info, id):
+    def get_node(cls, info, id):
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)